src/bosh-virtualbox-cpi/bin/test

```diff
@@ -5,14 +5,18 @@ result=0
 bin=$(dirname $0)
 
 echo -e "\n Formatting packages..."
-go fmt github.com/cppforlife/bosh-virtualbox-cpi/...
+go fmt bosh-virtualbox-cpi/...
 let "result+=$?"
 
 echo -e "\n Running build script to confirm everything compiles..."
 $bin/build
 $bin/build-linux-amd64
 let "result+=$?"
 
+echo -e "\n Running ginkgo tests..."
+ginkgo -r .
```

Comment:
Consider using `go run github.com/onsi/ginkgo/v2/ginkgo` here so that there isn't a need for a `ginkgo` binary to be installed in `$PATH`, and to ensure that the version used matches `go.mod`.

Example: https://github.com/cloudfoundry/bosh-agent/blob/main/bin/ginkgo#L5C1-L5C40